spec/factories/noticed_notifications.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
+arr1 = %w[ForumCommentNotification ForumThreadNotification]
+arr2 = %w[StarNotification ForkNotification NewAssignmentNotification NoType]
 FactoryBot.define do
   factory :noticed_notification do
-    type { "StarNotification" }
+    type { arr2.concat(arr1) }
```

Comment:
Type can be an array?

Reply:
Added default notification to be "StarNotification". Mentioned rest in a comment.